utils/categories/categoriesAggregateOptions.js

```diff
@@ -0,0 +1,57 @@
+const Subject = require('~/models/subject')
+const getRegex = require('~/utils/getRegex')
+
+const categoriesAggregateOptions = (query) => {
+  const { limit = 100, name = '', skip = 0 } = query
+
+  return [
+    {
+      $lookup: {
+        from: Subject.collection.name,
```

Comment:
just specify collection's name. Like this "from: 'subjects'"

Reply:
Fixed